### PR TITLE
11.0 mig mass editing

### PR DESCRIPTION
**Pull request by jnegrete**

### CODE REVIEW

File: mass_editing/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    'version': '11.0.1.0.0',
     'author': 'Serpent Consulting Services Pvt. Ltd., '
               'Tecnativa, '
               'Odoo Community Association (OCA)',
     'contributors': [
         'Oihane Crucelaegui <oihanecrucelaegi@gmail.com>',
         'Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>',
-        'Jay Vora <jay.vora@serpentcs.com>'
+        'Jay Vora <jay.vora@serpentcs.com>',
+        'Juan Negrete <jnegrete@casasalce.com>'
```
Comment (yajo) [inline mass_editing/__manifest__.py]: Contributors nowadays go into README, this part can be dropped from here.

File: mass_editing/hooks.py [hunk 2]
```diff
@@ -7,6 +6,5 @@ def uninstall_hook(cr, registry):
     cr.execute("""SELECT id FROM ir_act_window
                WHERE res_model = 'mass.editing.wizard'""")
     for res in cr.dictfetchall():
-        value = 'ir.actions.act_window,%s' % res.get('id')
-        cr.execute("DELETE FROM ir_values WHERE value = %s", (value, ))
+        cr.execute("DELETE FROM ir_act_window WHERE id = '%s'" % res.get('id'))
```
Comment (yajo) [inline mass_editing/hooks.py]: 1. Is this needed? Doesn't Odoo by itself automatically remove tables when an addon is uninstalled?
   - Yes: Add a docstring on why.
   - No: remove it.
2. This query is a little bit.... well, not KISS. Just `DELETE FROM ir_act_window WHERE res_model = 'mass.editing.wizard'` and you're done

File: mass_editing/wizard/mass_editing_wizard.py [hunk 3]
```diff
@@ -238,7 +238,7 @@ def create(self, vals):
                 self._context.get('active_ids')):
             model_obj = self.env[self._context.get('active_model')]
             values = {}
-            for key, val in vals.items():
+            for key, val in list(vals.items()):
```
Comment (yajo) [inline mass_editing/wizard/mass_editing_wizard.py]: why?

File: mass_editing/wizard/mass_editing_wizard.py [hunk 4]
```diff
@@ -208,7 +208,7 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
                             'colspan': '4',
                         })
             # Patch fields with required extra data
-            for field in all_fields.values():
+            for field in list(all_fields.values()):
```
Comment (yajo) [inline mass_editing/wizard/mass_editing_wizard.py]: why?

**Review by yajo — CHANGES_REQUESTED** [review]
Could you explain why are you removing all things related to `ir.values`?

Also, squash all translation-related commits together and fix Travis please.

**Comment by jnegrete** [comment]
Hello Jairo,

ir.values doesn't exist in odoo v11.0

I add the query and remove list, they were added by 2to3 -wnj4 --no-diffs

Regards.

2017-11-13 3:50 GMT-06:00 Jairo Llopis <notifications@github.com>:

File: mass_editing/__manifest__.py [hunk 5]
```diff
@@ -9,8 +9,7 @@
     'contributors': [
```
Comment (yajo) [inline mass_editing/__manifest__.py]: Please remove the key and move contributors to README

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by CasVissers-360ERP** [comment]
I ran a functional test everything worked fine untill I tried to edit some values:

```
TypeError: val is null

https://xxx.odoo.com/web/content/960-ac050f6/web.assets_backend.js:1245
Traceback:
_parseServerData/<@https://xxx.odoo.com/web/content/960-ac050f6/web.assets_backend.js:1245:545
_.forEach@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:12:558
_parseServerData@https://xxx.odoo.com/web/content/960-ac050f6/web.assets_backend.js:1245:288
_fetchRecord/<@https://xxx.odoo.com/web/content/960-ac050f6/web.assets_backend.js:1185:81
then/</</<@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:802:678
fire@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:796:281
fireWith@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:801:198
then/</</<@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:802:849
fire@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:796:281
fireWith@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:801:198
Deferred/</deferred[tuple[0]]@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:803:31
rpc/</<@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:3858:346
then/</</<@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:802:678
fire@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:796:281
fireWith@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:801:198
Deferred/</deferred[tuple[0]]@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:803:31
fire@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:796:281
fireWith@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:801:198
then/</</<@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:802:849
fire@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:796:281
fireWith@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:801:198
then/</</<@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:802:849
fire@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:796:281
fireWith@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:801:198
done@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:1192:86
callback@https://xxx.odoo.com/web/content/906-d57f445/web.assets_common.js:1212:15
```

**Comment by jnegrete** [comment]
Hi Cas Vissers,

Encontré un problema con los campos computados (like standard_price), puede
ser tu caso, prueba con el nuevo cambio

2017-11-15 6:42 GMT-06:00 Cas Vissers <notifications@github.com>:

**Comment by yajo** [comment]
Please remove cd3c87e, it was better before. If any computed method fails with that, it should be fixed in the computed method, not there.

OTOH, @CasVissers' logs are JS logs, not Python logs, so there's a high chance that the problem is in the client side.

As always, those logs are not in `?debug=assets` mode, so they are almost completely useless. Can you switch to that mode and paste the good logs please @CasVissers?

**Comment by RoelAdriaans** [comment]
Shoudn't this be moved to https://github.com/OCA/server-ux/ as per https://docs.google.com/spreadsheets/d/1Xg95cW4TFMf_Lo5i_CZC_qOOfN8RgxPRc0LJTLTkdUI/edit?usp=sharing ( From issue #1008 )

**Comment by pedrobaeza** [comment]
Yeah!

**Comment by yajo** [comment]
True, closing. Reopen there please.

**Pull request closed without merging**